config/test.yml

```diff
@@ -1,5 +1,5 @@
 database:
-    dsn: sqlite::memory:
+    dsn: "sqlite::memory:"
```

Comment:
At least NetBeans does say it's invalid. It could be that the : could interfere. And the symfony cookbook writes that with :, quotes are mandatory.

http://symfony.com/doc/current/components/yaml/yaml_format.html